setup.py

```diff
@@ -47,7 +47,7 @@
 # Take a look at https://www.python.org/dev/peps/pep-0440/
 # for a consistent versioning pattern.
 
-PSYCOPG_VERSION = '2.8.dev0'
+PSYCOPG_VERSION = '2.8.dev1'
```

Comment:
This is not necessary, and doesn't belong to this branch, please remove and rebase

Reply:
done